src/account/UpgradeableModularAccount.sol

```diff
@@ -305,7 +300,7 @@ contract UpgradeableModularAccount is
         onlyProxy
         wrapNativeFunction
     {
-        _upgradeToAndCallUUPS(newImplementation, data, true);
+        UUPSUpgradeable.upgradeToAndCall(newImplementation, data);
```

Comment:
Does this need to fully-qualified name to call? If it's an abstract contract / mixin and we're calling the `public` function internally, we should be able to just directly do:
```suggestion
        upgradeToAndCall(newImplementation, data);
```

right?

Reply:
good catch, fixed!